frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -178,7 +178,7 @@ describe('CRD extensions', () => {
       await yamlView.setEditorContent(safeDump(newContent));
       expect(yamlView.getEditorContent()).toContain(`location: ${location}`);
       await yamlView.saveButton.click();
-      await browser.wait(until.visibilityOf(crudView.successMessage), 1000);
+      await browser.wait(until.visibilityOf(crudView.successMessage), 10000);
```

Comment:
Do we need to increase the timeout here?

Reply:
I had to increase that time (perhaps not that much) on my system to get the test to consistently pass.  Shall I revert or lower the value?

Reply:
1s is pretty slow, i think it's ok. do you mind squashing?

Reply:
@spadgett - Reverted to 1s and squashed.